dbus_service.py

```diff
@@ -509,10 +496,8 @@ def is_data_up2date(self):
         if self.dtuvariant == constants.DTUVARIANT_AHOY:
             ts_last_success = self.get_ts_last_success(meter_data)
             age_seconds = time.time() - ts_last_success
-            logging.debug(
-                "is_data_up2date: inverter #%d: age_seconds=%d, max_age_ts=%d",
-                self.pvinverternumber, age_seconds, self.max_age_ts
```

Comment:
As I said above. For me, it looks squashed. sure there are two lines less but does it improve the readability?

Reply:
Hm. My opinion: Especially logging lines should require as little space as possible as they are only boilerplate.